### ISSUE TITLE
List position does not reset when changing software categories.

### ISSUE THREAD

**Issue by XV-02** [pop-os/cosmic-store#23]
On fe2f767, if I select a software category such as `Game` from the draw, and scroll half way through the presented options, and then change to another category such as `Develop`, the new list will be displayed at the half-way point, not at the head of the list.

Technically, it isn't actually at the same scrolled percentage, as moving from half-way through `Installed` - which may have 50 entries - to `Game` - over 1000 - has the list of game related packages scrolled some value through, but it is neither half-way nor ~3% of the way through the list (as it would be if the raw percentage or integer number of scrolled items was what counted respectively)

**Comment by git-f0x** [comment]
This behavior is currently present in all libcosmic apps (easiest to see in Settings, Files and Store).

**Comment by XV-02** [comment]
Good catch @git-f0x. I can reproduce it on my end, which suggests this is a libcosmic issue, so I'll move it over there.
